packages/elements/src/password-field/__test__/__snapshots__/password-field.test.snap.js

```diff
@@ -9,12 +9,19 @@ snapshots["password-field/PasswordField Default DOM structure and properties are
 >
 <ef-icon
   aria-label="Show password"
+  aria-pressed="false"
   icon="eye"
   part="icon"
   role="button"
   tabindex="0"
 >
 </ef-icon>
+<div
+  aria-live="polite"
+  class="visually-hidden"
```

Comment:
Use `aria-label` might be a better solution.

Reply:
password-field originally use `aria-label`. `aria-live` seems to be the only option get VoiceOver announced toggle state during focus and toggle activation.

Reply:
I mean, we can toggle a password to attribute `aria-label` and place `aria-live` in the same `div` or `span` which the class `visually-hidden` might be removed.
```html
<span aria-label="password" aria-live="polite"></span>
```

The other option we might be use `<hidden />` and toggle text inside with `aria-live`

Reply:
stick with the current implementation as using both `aria-label` and `aria-live` would trigger a bug on NVDA side. Not to mention that these 2 attributes are not expected in the same element.